doc/source/conda_forge_guide.rst

```diff
@@ -86,13 +93,54 @@ Step 2. Upload ``meta.yaml``
 10. After the CI passes, create a new comment: ``@conda-forge/help-python Hello Team, ready for review!``
 
 Step 3. Wait for review and merge
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. Wait for a ``conda-forge`` reviewer to review your submission (may take up to one week)
+1. Wait for a ``conda-forge`` volunteer reviewer to review your submission. It may take up to one week.
 
 2. Once the PR is merged by the reviewer (1) your package is available on conda-forge, and (2) a new repository will be created under https://github.com/conda-forge/package-name-feedstock/. Example: https://github.com/conda-forge/diffpy.structure-feedstock.
 
-## conda-forge: pre-release
+
+I already have a conda package. How do I distribute a new package version?
```

Comment:
migrating content from `release_guide.rst` - we shall combine all content related to conda-forge under `conda_forge_guide.rst`